### DISCUSSION TITLE
Can we play as IND/OPFOR against BLUFOR

### DISCUSSION THREAD

**Discussion by ItzRa1nY** [Vdauphin/HeartsAndMinds#1623] [Question and answer] [answered]
can we make it so we go against the BLUFOR faction as IND/OPFOR (BLUFOR as player and IND/OPFOR as limited players and ai )

**Comment by Vdauphin** [comment, marked as answer]
Hello,

Yes see http://vdauphin.github.io/HeartsAndMinds/faction_player

Cheers

**Reply by Oldman-BR** [reply]
Vdauphin, i did the procedure of the link you sent to change the faction. In the mission that im editing, the mod im using as my main faction came as independent and the enemy is OPFOR. The respawn is blocked. I changed the side in the mission.sqf line 463 (btc_player_side = independent;) and the line 464 (btc_respawn_marker = "respawn_independent";), changed the name of the marker in the editor and the respawn still blocked. Can you tell me what is the problem please? Thanks.

**Reply by Vdauphin** [reply]
Hello,

Please use only marker documented https://community.bistudio.com/wiki/Arma_3:_Respawn#Respawn_Types

Cheers

**Reply by Oldman-BR** [reply]
SO, please what do i have to put in the line 463 to set the player side?

**Reply by Vdauphin** [reply]
You need to choose the player side you want to play
See https://community.bistudio.com/wiki/Side

**Reply by Oldman-BR** [reply]
Solved thanks. in the line 463 must be as btc_player_side = independent; and the line 464 as btc_respawn_marker = "respawn_guerrila";

Have to change the marker's name in the edtor to respawn_guerrila. Thanks man.
